scripts/exregional_run_analysis.sh

```diff
@@ -643,7 +705,130 @@ if [ $netcdf_diag = ".true." ]; then
 fi
 
 done
+#
+#-----------------------------------------------------------------------
+#
+# when running observer, loop over the rest of members to get GSI diag*
+#
+#-----------------------------------------------------------------------
+#
+
+if [ ${if_observer} == "True" ]; then
+  mv gsiparm.anl gsiparm.anl_mem0001
+  ensmem=2
+
+  while [[ $ensmem -le ${NUM_ENS_MEMBERS} ]]; do
+    echo "Running observer for member $ensmem "
+    miter=0
+    lread_obs_save=.false.
+    lread_obs_skip=.true.
+    rm -f pe0*
+    memid=$(printf "%04d" $ensmem)
+    bkpath=${cycle_dir}/mem$ensmem/fcst_fv3lam/RESTART
+    restart_prefix=${YYYYMMDD}.${HH}0000.
+    checkfile=${bkpath}/${restart_prefix}fv_core.res.tile1.nc
+    if [ -r "${checkfile}" ]; then 
+      print_info_msg "$VERBOSE" "Found ${checkfile}; Use it as background for analysis "
+      ln_vrfy  -snf  ${bkpath}/${restart_prefix}fv_core.res.tile1.nc             fv3_dynvars
+      ln_vrfy  -snf  ${bkpath}/${restart_prefix}fv_tracer.res.tile1.nc           fv3_tracer
+      ln_vrfy  -snf  ${bkpath}/${restart_prefix}sfc_data.nc                      fv3_sfcdata
+      fv3lam_bg_type=0
+    else
+      print_err_msg_exit "$VERBOSE" "Error: cannot find background: ${checkfile}"
+    fi
+
+. ${USHDIR}/templates/gsiparm.anl.sh
```

Comment:
Line 740 should be:
. ${FIX_GSI}/gsiparm.anl.sh

Reply:
Done. Thanks!